react/react-training/src/app/components/AppRoutes/index.js

```diff
@@ -31,9 +34,7 @@ const mapStateToProps = store => ({
   userIsLogged: store.LoginReducer.userIsLogged
```

Comment:
`userIsLogged: store.LoginReducer.userIsLogged` applying that approach mentioned above